server/src/main/kotlin/fi/oph/kitu/yki/suoritukset/YkiSuoritusEntity.kt

```diff
@@ -44,7 +44,7 @@ data class YkiSuoritusEntity(
     val tarkistusarvioinninSaapumisPvm: LocalDate?,
     val tarkistusarvioinninAsiatunnus: String?,
     val tarkistusarvioidutOsakokeet: Int?,
-    val arvosanaMuuttui: Boolean?,
+    val arvosanaMuuttui: Int?,
```

Comment:
Tätä vois jollain tavalla dokumentoida, että se muodostuu bittimaskista `puhe|kieliosaaminen|tekstin_ymmärrys|puheen_ymmärrys`, esim:

```suggestion
    /**
    * Tells whether the grade have changed. It is a bit mask value:
    * 1st bit: puhe
    * 2nd bit: kieliosaaminen
    * 3rd bit: tekstin ymmärrys
    * 4th bit: puheen ymmärrys
    */
    val arvosanaMuuttui: Int?,
```

Reply:
Joo, vois olla hyvä, paitsi eikö tää mee päinvastaisessa järjestyksessä? Jos "1=pu, 2=ki, 4=ty, 8=py" niin 0001=puhe, 0010=kirjoittaminen, 0100=tekstin ymmärtäminen ja 1000=puheen ymmärtäminen?

Reply:
Laitan tähän kommentin vaan silleen, miten rajapintakuvauksessa se on laitettu. Voidaan sit myöhemmin miettiä et miten tää parhaiten kannattaa käsitellä pitkällä aikavälillä.